APE0.rst

```diff
@@ -0,0 +1,393 @@
+  Committee will put it to a vote among active Voting Members. Votes for adding
+  voting members will be carried out at most twice a year to avoid too many
+  elections. The voting dates will be announced with at least four weeks notice,
+  and additional nominations will be accepted until two weeks before the
+  election date. The voting will be open for at least two weeks. To successfully
+  gain an appointment as a Voting Member, the nominee must receive at least four
+  positive votes, as long as that constitutes a majority of those voting.
+* The candidate will be informed promptly at the close of voting by a
+  Coordination Committee member. If the vote is not affirmative, the
+  Coordination Committee will provide feedback to the nominee.
+
+At least one round of voting for adding voting members should be carried out
+before the initial election of the Coordination Committee.
+
+Term and Active/Emeritus Status
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Voting Members have no term or term limits. Voting Members who have stopped
+contributing are encouraged to declare themselves as *emeritus*. Those who have
+not made any significant contribution for two years may be asked to move
+themselves to the *emeritus* category by the Coordination Committee. If no
+response is received, the Coordination Committee may automatically change a
+Voting Member’s status to *emeritus*. To record and honor their contributions,
+*emeritus* Voting Members will continue to be listed. *Emeritus* Voting Members
+are not able to participate in votes.
+
+Removing Voting Members
+^^^^^^^^^^^^^^^^^^^^^^^
+In exceptional circumstances, it may be necessary to remove someone from the
+Voting Members against their will. A vote must be held to remove a Voting
+Member. Such a vote is triggered by a motion made by an active Voting Member,
+which must be seconded by an additional Voting Member. The vote must conclude
+no more than three months after the motion is seconded. Removal requires
+approval by two-thirds of all active Voting Members at the time the motion is
+made. The motion, second, and vote will be held in private. Removal under this
+provision will be reflected by updating the list of Voting Members.
+
+It may be necessary for the Ombudsperson and the Coordination Committee to
+remove a Voting Member for violations of the Code of Conduct. In this case, the
+Coordination Committee and Ombudsperson will work together to make this
+decision.
+
+Approving and Modifying This Charter
+------------------------------------
+This document was submitted following the process in `APE 1`_, and the normal APE
+acceptance procedures will be followed. The Coordination Committee at the time
+of submitting this APE are all co-authors and therefore will not override any
+consensus of the community on accepting the final version.
+
+Changes to this Charter after it has been accepted should follow the
+modification process in `APE 1`_, with the exception that the final approval of
```

Comment:
So, this this PR blocked by #60 ?

Reply:
I think not in the sense that right now there *is* no APE1 modification process.  You're right the intent was for #60 to go in first, but I think we can de-couple thme by just saying this one is saying "whatever the process is, it applies" (which means it is implicitly updated when #60 is in)